### PR TITLE
add type checking string and new error

### PR DESCRIPTION
**Pull request by mdchad**
I made a small change to check if the coordinates were string. In the future user will get an error if they pass in a string number instead of number

Another change is to throw error instead of returning empty string since empty string does not point to a direction. This will help user to find bug easier

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ const PLATFORM = Platform.OS;
 
 export const OpenMapDirections = (frmCoord = null, toCoord, transportType) => new Promise((resolve, reject) => {
 	const _frmCoord = _checkParameters(frmCoord) !== null ? `?saddr=${_checkParameters(frmCoord)}` : '';
-	const _toCoord = _checkParameters(toCoord) !== null ? `&daddr=${_checkParameters(toCoord)}` : '';
+	if (_checkParameters(toCoord) !== null) {
+		const _toCoord = `&daddr=${_checkParameters(toCoord)}`
```
Comment (laki944) [inline index.js]: Now you isolate const _toCoord in if statement so you can't use in the line 14.

**Review by laki944 — CHANGES_REQUESTED** [review]
Idea is ok, but first fix issue with _toCoord and after that i will merge pull request.

**Comment by mdchad** [comment]
My mistake. I have amended the const declaration to the top.

**Review by laki944 — APPROVED** [review]
Now it's OK. Thanks for PR.

**Pull request merged**